server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -322,6 +338,9 @@ public void writeTo(StreamOutput out) throws IOException {
         if (out.getVersion().onOrAfter(Version.V_1_2_0)) {
             out.writeOptionalWriteable(shardIndexingPressureStats);
         }
+        if (out.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
Lets directly add V_2_4_0 and follow this up with a backport

Reply:
Updated!
Created a backport PR as well – https://github.com/opensearch-project/OpenSearch/pull/5039